fanout/create_group_fanout.js

```diff
@@ -0,0 +1,325 @@
+const asyncAuto = require('async/auto');
+const asyncMap = require('async/map');
+const {broadcastTransaction} = require('ln-sync');
+const {getChainBalance} = require('ln-service');
+const {getIdentity} = require('ln-service');
+const {getMethods} = require('ln-service');
+const {getUtxos} = require('ln-service');
+const {getNodeAlias} = require('ln-sync');
+const {returnResult} = require('asyncjs-util');
+const tinysecp = require('tiny-secp256k1');
+
+const assembleFanoutGroup = require('./assemble_fanout_group');
+
+const descriptionForGroup = 'group fanout';
+const asBigUnit = n => (n / 1e8).toFixed(8);
+const asOutpoint = utxo => `${utxo.transaction_id}:${utxo.transaction_vout}`;
+const {isArray} = Array;
+const isNumber = n => !isNaN(n);
+const isPublicKey = n => !!n && /^0[2-3][0-9A-F]{64}$/i.test(n);
+const isOdd = n => !!(n % 2);
+const isValidMembersCount = (n, count) => !n.length || n.length === count - 1;
+const join = arr => arr.join(', ');
+const maxGroupSize = 420;
```

Comment:
I think this should be lowered because you can have multiple outputs per user

How about 42 max?